### DISCUSSION TITLE
What's up with extract_regex  and content

### DISCUSSION THREAD

**Discussion by AntonioSun** [smicyk/groovy-jmeter#94] [Q&A] [answered]
https://github.com/smicyk/groovy-jmeter/blob/0fe1b913c9c0094c101a05c8a9ae983f7b047c15/examples/distributed/script.groovy#L32

I don't see `content` in https://github.com/smicyk/groovy-jmeter/wiki/reference#extract_regex.

Where is it coming from?

**Comment by smicyk** [comment, marked as answer]
The __expression__ is '"trackId", content="([0-9]+)"', the content is not property it is just a part of expression we are looking in the response.
